dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/entity/WorkFlowResponseTotalList.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Apache Software Foundation (ASF) under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Apache Software Foundation (ASF) licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.dolphinscheduler.api.test.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashMap;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class WorkFlowResponseTotalList {
+
+    private String code;
+
+    private Date createTime;
+
+    private String description;
+
+    private String executionType;
+
+    private String flag;
+
+    private Arrays globalParamList;
+
+    private HashMap globalParamMap;
```

Comment:
It's better to define generic types.

Reply:
do you mean `Map` instead of `HashMap`, or just `String` and add method to convert string to map?